validator/src/main.rs

```diff
@@ -1846,6 +1847,12 @@ pub fn main() {
         BlockProductionMethod
     )
     .unwrap_or_default();
+    validator_config.transaction_struct = value_t!(
+        matches, // comment to align formatting...
```

Comment:
copied from `block_production_method`.
Think to just make all the various block_{verification|production} options consistently multi-line instead of some being 1 line, others not.

@ryoqun IIRC you added this, idk if you feel strongly about keeping this or if we should remove it?

Reply:
> Think to just make all the various block_{verification|production} options consistently multi-line instead of some being 1 line, others not.

yeah, this is what i intentionally added this. lol

> @ryoqun IIRC you added this, idk if you feel strongly about keeping this or if we should remove it?

no, strong opinion. you can remove it if you're okay with inconsistent formatting.

Reply:
I'm fine leaving as is. Just thought it was funny